motr/idx_dix.c

```diff
@@ -1267,7 +1294,10 @@ static int dix_index_lookup(struct m0_op_idx *oi)
 	int             rc;
 
 	M0_ASSERT(dix_iname_args_are_valid(oi));
-	rc = dix_mreq_create(oi, &req);
+	if (SKIP_CROW_IDX_OPS) 
+		rc = dix_req_create(oi, &req);
```

Comment:
Change this to,
rc = SKIP_CROW_IDX_OPS ? dix_req_create(oi, &req) :  dix_mreq_create(oi, &req);